fluent-react/src/localized.js

```diff
@@ -111,7 +112,15 @@ export default class Localized extends Component {
       }
     }
 
-    // If the message has a null value, we're onl interested in its attributes.
+    // If the wrapped component is a known void element, explicitly dismiss the
+    // message value and do not pass it to cloneElement in order to avoi the
+    // "void element tags must neither have `children` nor use
+    // `dangerouslySetInnerHTML`" error.
+    if (elem.type in VOID_ELEMENTS) {
+      return cloneElement(elem, localizedProps);
```

Comment:
any chance we'd want to log a warning to the console to indicate an invalid message format? Or do we prefer silently handling the error?

Reply:
I'd like to tackle the problem of error reporting in a separate PR.